### ISSUE TITLE
best loss ?

### ISSUE THREAD

**Issue by rajarsheem** [unilight/R-NET-in-Tensorflow#5]
Can you share the best train/val loss you observed ?

**Comment by unilight** [comment]
I don't think there's a best "training loss" since you can always train forever and get a really low training loss. Anyway, after training for 28 epochs (the same number of epochs I trained to obtain the model weights I released), the training loss was 3.05.

As for the validation loss, I didn't really keep track of that. I think you can always check the EM/F1 on dev set to see if you have overfitted the data.
